portal-web/test/functional/com/liferay/portalweb/tests/enduser/abtest/ABTesting.testcase

```diff
@@ -108,9 +108,33 @@ definition {
 	@ignore = "true"
 	@priority = "3"
 	test AssertGoalInputWhenEdit {
+		task ("Create AB test, create variant and run the test") {
+			ABTest.createABTest(
+				goal = "Click",
+				testTitle = "AB test title");
+
+			ABTest.createVariant(variantName = "Variant name");
+		}
+
+		task ("Edit test A/B and assert that user tries to edit one AB Test") {
+			ABTest.editOrDeleteABTest(value = "Edit");
 
-		// TODO LPS-137479 AssertGoalInputWhenEdit pending implementation
+			AssertElementPresent(locator1 = "ABTest#CHANGE_SELECT_GOAL");
 
+			Select(
+			locator1 = "ABTest#CHANGE_SELECT_GOAL",
+			value1 = "Bounce Rate");
+			
+			Click(locator1 = "ABTest#SAVE_AB_TEST_BUTTON");
+
+			Alert.viewSuccessMessage();
+
+			ABTest.runTest();
+		}
```

Comment:
This test is more simple than that. It is not needed to change the AB test goal. The poshi test goal is to open the Edit modal and check that the goal of the AB test (in this case "Click") is present in the modal. After that, you can just close the modal and terminate the test :)

Reply:
Ok, thank you Carlos, I was doubt whether it was necessary or not. i'll fix it too.